test/models/commitment_test.rb

```diff
@@ -8,4 +8,23 @@ class CommitmentTest < ActiveSupport::TestCase
   test "does not save an invalid commitment" do
     assert_not commitments(:invalid_commitment).save
   end
+
+  test "does not save a live commitment with joint governance unless it has a joint_governance_description" do
```

Comment:
You switched to single quotes in the previous test, but are using double quotes here

Reply:
i've switched back to double quotes, as that's what the rails generator uses. personally, i prefer single quotes apart from when we need double quotes for string interpolation, but that's just preference